### DISCUSSION TITLE
best way to have « » around Quoted(DoubleQuote) instead of “ ” when exporting to DOCX

### DISCUSSION THREAD

**Discussion by massifrg** [jgm/pandoc#9976] [Q&A] [answered]
I'm converting pandoc JSON to docx.

Quoted(DoubleQuote) inlines are rendered between English double quotes in docx.

I'd like to render them between « » (guillemots).

I know how to write a lua filter that puts `Str("«")` and `Str("»")` around the contents of a `Quoted(DoubleQuote)`, while removing the `Quoted` container.
But maybe someone knows a better strategy:

- is there a Word setup to automatically have guillemots around double quotes? (language setup? guillemots are used in French, but also in Italian, that is the language of my documents)

- is there a pandoc option that I can use? (I searched the user guide, but found nothing about that)

**Comment by tarleb** [comment, marked as answer]
It's not quite what you're asking for, but the [`pandoc-quotes`](https://github.com/odkr/pandoc-quotes.lua) filter is rather nice.

**Comment by massifrg** [comment]
Thank you, @tarleb 

that's the sort of filter I was going to write, though `pandoc-quotes` is much, much better.

Since docx is an end format, not the one for semantic storage, the removal of the `Quoted` semantic marking is not an important loss.
